treeherder/webapp/api/performance_data.py

```diff
@@ -840,6 +850,18 @@ def list(self, request):
                     'new_stddev_pct': new_stddev_pct,
                     'base_retriggerable_job_ids': base_grouped_job_ids.get(base_sig_id, []),
                     'new_retriggerable_job_ids': new_grouped_job_ids.get(new_sig_id, []),
+                    'graphs_link': PerformanceSummary.createGraphsLinks(
+                        {
+                            'originalProject': base_repository_name,
+                            'newProject': new_repository_name,
+                            'originalRevision': base_revision,
+                            'newRevision': new_revision,
+                        },
+                        [{'title': 'subtests', 'to': './comparesubtests%s' % urlencode(params)}],
```

Comment:
You don't need this code for the subtests link. I would suggest starting fresh without these methods, just add one method that receives the params mentioned in the earlier comment.